windows/ReactTestApp/Manifest.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+#include <list>
+#include <map>
+#include <optional>
+#include <string>
+
+#include <nlohmann/json.hpp>
+
+using json = nlohmann::json;
+using namespace std;
+
+namespace winrt::ReactTestApp::implementation {
```

Comment:
This should just be in its own namespace.
```suggestion
namespace ReactTestApp
 {
```